src/grandorgue/GOOrganController.cpp

```diff
@@ -1160,3 +1168,8 @@ GOLabelControl *GOOrganController::GetTemperamentLabel() {
 GOMainWindowData *GOOrganController::GetMainWindowData() {
   return &m_MainWindowData;
 }
+
+/**
+ * Return the Timer Manager for Metronome, Midi recorder, ...
+ */
+GOTimer *GOOrganController::GetTimerManager() { return m_TimerManager; }
```

Comment:
Usually Oleg wants such small and simple methods to have their implementation in the header file.